src/main/java/net/sf/mzmine/modules/peaklistmethods/identification/sirius/PeakListIdentificationTask.java

```diff
@@ -84,10 +85,18 @@
     fingerCandidates =
         parameters.getParameter(PeakListIdentificationParameters.CANDIDATES_FINGERID).getValue();
 
+    String massListName = parameters.getParameter(SingleRowIdentificationParameters.MASS_LIST).getValue();
+    List<String> massLists = MassListComponent.getMassListNames();
```

Comment:
This is not correct - you must check each scan separately when retrieving the mass list data for that scan. For example if scan 1 has a mass list called "MSL1" it doesn't guarantee that scan 2 also has a mass list of that name.